### PR TITLE
Az.DnsResolver Added default DNS port  

### PR DESCRIPTION
**Pull request by guandali**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Review by muwaqar — APPROVED** [review]

File: src/DnsResolver/Az.DnsResolver.psd1 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @{
   GUID = '88e239b3-f615-4018-adf3-f0952a0f46a8'
   RootModule = './Az.DnsResolver.psm1'
-  ModuleVersion = '0.1.2'
+  ModuleVersion = '0.1.3'
```
Comment (VeryEarly) [inline src/DnsResolver/Az.DnsResolver.psd1:4]: please do not update this version

File: src/DnsResolver/readme.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ require:
 input-file:
   - $(this-folder)/resources/specification/dnsresolver/resource-manager/Microsoft.Network/preview/2020-04-01-preview/dnsresolver.json
 
-module-version: 0.1.2
+module-version: 0.1.3
```
Comment (VeryEarly) [inline src/DnsResolver/readme.md:78]: please do not update this version

Comment (guandali) [reply src/DnsResolver/readme.md]: Updated

**Comment by VeryEarly** [comment]
Hi @guandali , please sync your branch with generation to have CI fixed

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**